lib/neo4j/shared/enum.rb

```diff
@@ -53,18 +53,29 @@ def enum(parameters = {})
 
       protected
 
-      def normalize_key_list(enum_keys)
+      def normalize_key_list(enum_keys, options)
+        case_sensitive = options[:_case_sensitive]
+        case_sensitive = Neo4j::Config.enums_case_sensitive if case_sensitive.nil?
```

Comment:
You could do

```ruby
case_sensitive = options[:_case_sensitive] || Neo4j::Config.enums_case_sensitive
```

or

```ruby
options.fetch(:_case_sensitive, Neo4j::Config.enums_case_sensitive)
```

Reply:
The difference between the two is that the first would fall back to `Neo4j::Config.enums_case_sensitive` if you get a `nil` value from `options[:_case_sensitive]` regardless of if there was no key or if there was a key but it just happened to be `nil`.  In the second one if `options[:_case_sensitive]` is set to `nil` then the result will be `nil` because `fetch` only falls back to the default when the key isn't set in the hash.

Reply:
I'll update it to use `fetch`, the first one doesn't quite work because if `options[:_case_sensitive] = false` and `Neo4j::Config.enums_case_sensitive = true`, then `case_sensitive = true`, when it should equal `false`.